php/PhpJsonApiSamples/Default.html

```diff
@@ -31,41 +31,45 @@
 	<form id="form1" runat="server">
 	<table style="width:40%">
 		<tr><th>Select any option </th></tr>
-		<!--<tr> 
-			<td><a href="ClientSideEncryptionDemo.aspx">ClientSide Encryption </a> </td>
-		</tr>   -->
+		<tr> 
+                        <!-- Demo1 checks the client side validations and then submit the valid form with PayTrace client side library  -->
```

Comment:
And maybe the verbiage could have more info. I don't know like, it doesn't necessary have to be client validation that I want to perform first, it could be any client side implementation I want to perform before submitting the form. The client side validation is just one example that I could perform before form submission.